.ci/buildAll.sh

```diff
@@ -3,7 +3,7 @@
 # Find paths that contain the Gradle wrapper executable.
 APP_PROJECTS=$(find $PWD -maxdepth 1 -type d -exec [ -x {}/gradlew ] \; -print -prune)
 
-pushd . > /dev/null
+root="$PWD"
```

Comment:
The variable should probably not be called `root`, because it's not necessarily the root of something. It's just the current directory when the script was started. Maybe `ORIG_PWD`?